locales/base/translation.json

```diff
@@ -2120,7 +2120,7 @@
     "importButton": "Import"
   },
   "sendSelectRecipient": {
-    "searchText": "Search by name, phone, wallet...",
+    "searchText": "Search by wallet address",
```

Comment:
ah, since we're not integrated with Crowdin here, we should also make the changes in the translated flows for consistency? google translate probably good enough there?